server/views/pages/prisons/sessionTemplates/editSingleSessionTemplate.njk

```diff
@@ -325,6 +325,23 @@
           <p>There are no location groups set up for this prison.</p>
         {% endif %} #}
 
+        {{ govukCheckboxes({
+          name: "showPublicServices",
+          fieldset: {
+            legend: {
+              text: "Show in public services?",
+              classes: "govuk-fieldset__legend--m"
+            }
+          },
+          errorMessage: errors | findError('showPublicServices'),
+          items: [
+            {
+              text: "Yes",
+              value: "yes",
+              checked: formValues.showPublicServices === 'Yes'
```

Comment:
Should this be `yes` instead of `Yes` - i.e. the form **value** rather than text label?

Reply:
no because the form value is built with an IF statement, that sets the value to 'Yes' - so all pages check this value the same way (Yes is used for the display on view single session template page)

Reply:
Hmm... not sure. Tried it locally and there's something not quite right. Possibly related to mixing yes/no cases and/or something else.

To reproduce:
* Go to 'Add session template' page
* **Uncheck** the 'show in public service' option
* Try to add the template - it will fail with validation errors because no other data filled in
* **BUT** the 'show in public services' has forgotten my choice: it's re-checked itself!

Same happens when updating an existing template.